docs/spec.md

```diff
@@ -51,6 +57,8 @@ Each JSON message must be an object containing a field called `op` which identif
 - `name`: free-form information about the server which the client may optionally display or use for debugging purposes
 - `capabilities`: array of strings, informing the client about which optional features are supported
   - `clientPublish`: Allow clients to advertise channels to send data messages to the server
+  - `parameters`: Allow clients to get & set parameters
+  - `parameterSubscribe`: Allow clients to subscribe to parameter changes
```

Comment:
Should we call it just `parameter` to be inline with `parameterSubscribe` ?

```suggestion
  - `parameter`: Allow clients to get & set parameters
  - `parameterSubscribe`: Allow clients to subscribe to parameter changes
```

Reply:
Or maybe `parameterSubscribe` should be `parametersSubscribe`. You make the call, change one of them so they're consistent.